quill-ndbc-postgres/src/main/scala/io/getquill/context/ndbc/PostgresEncoders.scala

```diff
@@ -26,6 +33,8 @@ trait PostgresEncoders extends LowPriorityPostgresImplicits with io.getquill.dsl
       if (v == null) ps.setNull(idx)
       else f(ps)(idx, v.map(ev).toArray[U])
 
+  implicit override def anyValMappedEncoder[I <: AnyVal, O](implicit mapped: MappedEncoding[I, O], encoder: Encoder[O]): Encoder[I] = mappedEncoder
```

Comment:
The trick here is that `anyValMappedEncoder` is in the primary class `PostgresEncoders` which is the 2st place the implicits get looked up and the `mappedEncoder` is an attached trait which gets looked up later. Also, `anyValEncoder` is part of `LowPriorityImplicits`. By attaching `LowPriorityImplicits` to `LowPriorityPostgresImplicits` I made `anyValEncoder` even later in the implicit search order then `mappedEncoder` or `anyValMappedEncoder`. At least that's how I think it works...

Reply:
Thank you very much for your help here :)